documentation/Views.md

```diff
@@ -0,0 +1,43 @@
+# Views
+
+SIMPLE offers several database views for ease of searching. 
+A database view is a virtual table that is defined by a query. 
+It does not store any data itself, but rather provides a way to access data from one or more tables in a database.
+
+Because views are not part of the standard SIMPLE schema, they must be activated when using `astrodbkit2`. 
+
+```python
+import sqlalchemy as sa
+from astrodbkit2.astrodb import Database, and_, or_
+
+# Establish connection to database; your connection string may be different
+connection_string = 'sqlite:///SIMPLE.db'
+db = Database(connection_string)
+
+# Inspect views in the database
+insp = sa.inspect(db.engine)
+
+# Examine available views
+print(insp.get_view_names())  # list of available views
+print(insp.get_view_definition('ParallaxView'))  # view definition
+
+# Create object reflecting the view
+# DO NOT USE db.metadata or you will treat the view as a real table 
+# and break other astrodbkit2 functionality
```

Comment:
I don't know if this warning about db.metadata is useful or if it may cause further confusion. It may be something to remove.

Reply:
I'll believe I'll keep it for more information/visibility.